### DISCUSSION TITLE
Marine Hydrokinetic Wave

### DISCUSSION THREAD

**Discussion by tumibytes** [NREL/ReEDS-2.0#103] [Q&A: Features]
Hi, ReEDS Team! I read in the documentation that ReEDS does have a representation for MHK Wave Technologies, but has not been used in any recent work. Is there a way that I can utilize this capability for research? Are there certain files in the inputs folder I can look through for more information? I know that you can add technologies in the generatordata.py script in Augur, but I was unsure if this was the proper way to go about using MHK Wave Tech. Are there any tutorials that are out there I may reference? Please let me know! Thank you!

**Comment by vcelloho** [comment]
We had MHK as a technology in the Heritage version of ReEDS and the technology sets were preserved for ReEDS 2.0. The underlying data being a decade out of date was not ported over.

For your effort are you looking to add data for MHK using your own datasets? Are you interested in obtaining the prior ReEDS Heritage implementation of MHK? The ReEDS heritage data would require some work to integrate with the current version of the model.

**Reply by tumibytes** [reply]
We are looking to add data for MHK using datasets that we found. Would editing the generatordata.py script to include MHK technology be the appropriate way to start? Is there a method in the “How to Add Inputs into the Augur Module of ReEEDS” video that can be of use here? What further steps should we take for our efforts? Are there supporting materials to guide a user on how to customize their own simulations, including adding/removing generator/loads, changing energy price, load profiles, etc.?

**Comment by rebeccamccabe** [comment]
@etumibay, I also am interested in potentially using ReEDS to investigate wave energy. Can you share how your project worked out, were you able to integrate your datasets into the new version of ReEDS? Any chance your code is open-source, or do you have any writeups of your results?
For reference, I found what appears to be the original study that used MHK in ReEDS over a decade ago: http://www.ourenergypolicy.org/wp-content/uploads/2012/10/The-Future-of-Wave-Power-MP-9-20-12.pdf
Thanks!

**Reply by tumibytes** [reply]
Hi, @rebeccamccabe . For my internship, we changed the focus of my project and thus did not do further work to integrate the MHK datasets into ReEDS. I apologize I couldn't be of more help.
